QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeFull.java

```diff
@@ -40,10 +40,12 @@ protected void runAutonomous() throws InterruptedException,
 		turn(90);
 		drive(autoZoneDistance.get());
 		// Drop everything off
-		// TODO: Put down container
+		setClampHeight(0.0f);
+		setClampOpen(true);
+		setClampHeight(1.0f);
 		DriveCode.octocanumShifting.set(true);
 		strafe(1.0f, strafeTime.get());
-		// TODO: Put down container
+		// TODO: Put down top container
 		strafe(1.0f, strafeTime.get());
 		ejectTotes();
 		DriveCode.octocanumShifting.set(false);
```

Comment:
You will probably want to eject the totes before setting down the second container, if possible.